client/app/services/custom-button/custom-button.component.js

```diff
@@ -26,6 +26,9 @@ function CustomButtonController ($state, EventNotifications, CollectionsApi, RBA
   }
 
   function invokeCustomAction (button) {
+    if (!button.enabled) {
```

Comment:
rather than an empty return consider wrapping the function body in an `if(button.enabled)`